tests/Unit/Domain/CoordinateMaps/Test_Wedge2D.cpp

```diff
@@ -0,0 +1,136 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+#include <random>
+
+#include "Domain/CoordinateMaps/Wedge2D.hpp"
+#include "Utilities/StdHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D", "[Domain][Unit]") {
+  // Set up random number generator:
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_int_distribution<> int_dis(0, 3);
+  std::uniform_real_distribution<> real_dis(-1, 1);
+  std::uniform_real_distribution<> inner_dis(1, 3);
+  std::uniform_real_distribution<> outer_dis(4, 7);
+
+  // Our default approx value is too stringent for this test
+  Approx local_approx = Approx::custom().epsilon(1e-10);
```

Comment:
I'm surprised by how loose this tolerance is. Is this really the inherent error in the mapping? Do we understand why?

Reply:
I've investigated and it turns out to just be checking the jacobian, which is checked against a numerical derivative. I'm refactoring the testing of the jacobian into its own function anyway so we can switch to using the usual `approx` here and use the less stringent one inside of that function.